pkg/ddl/ingest/backend_mgr.go

```diff
@@ -94,29 +93,16 @@ func (b *BackendCtxBuilder) ForDuplicateCheck() *BackendCtxBuilder {
 var BackendCounterForTest = atomic.Int64{}
 
 // Build builds a BackendCtx.
```

Comment:
mention that `BackendCtxBuilder` does not own `bd *local.Backend` so it will not close it when `BackendCtxBuilder` closes. Caller should close `bd` instead.

Reply:
https://github.com/pingcap/tidb/pull/59165/commits/fa3f43d7bee14e46e047e29d8c187d2a276dc52c